src/common/ContractInteractor.ts

```diff
@@ -379,7 +379,7 @@ export default class ContractInteractor {
   }
 
   async estimateRelayTransactionMaxPossibleGas (relayRequest: RelayRequest, signature: PrefixedHexString): Promise<number> {
-    const maxPossibleGas = await web3.eth.estimateGas({
+    const maxPossibleGas = await this.web3.eth.estimateGas({
```

Comment:
You already have a method to calculate gas inside this class, you could use `await this.estimateGas` instead of `await this.web3.eth.estimateGas`

Reply:
agreed

Reply:
Done